frontend/app/routes/activate.js

```diff
@@ -0,0 +1,26 @@
+import Route from "@ember/routing/route";
+import fetch from "fetch";
+
+export default Route.extend({
+  model({ uid, token }) {
+    const data = {
+      data: {
+        type: "activation-views",
+        attributes: {
+          uid,
+          token
+        }
+      }
+    };
+
+    return fetch("/api/v1/auth/activate", {
+      method: "post",
+      dataType: "json",
```

Comment:
`dataType` is an option of `$.ajax`. With `fetch` you can just set the `Content-Type` header which you are doing. This line is obsolete.